docs/docs/concepts/cloud-agent.mdx

```diff
@@ -0,0 +1,49 @@
+---
+id: cloud-agent
+title: Tracetest Cloud Agent (Beta)
+description: Tracetest Cloud Agent is a lightweight, dependency-free program that runs within the Tracetest Cloud Infrastructure.
+keywords:
+  - tracetest
+  - trace-based testing
+  - observability
+  - distributed tracing
+  - testing
+  - agent
+  - cloud agent
+  - serverless agent
+image: https://res.cloudinary.com/djwdcmwdz/image/upload/v1698686403/docs/Blog_Thumbnail_14_rsvkmo.jpg
+---
+
+Tracetest Cloud Agent is a lightweight, dependency-free program that runs within the Tracetest Cloud Infrastructure. Allowing you to trigger tests against public endpoints while collecting traces from public data stores or listening for incoming OTLP data.
```

Comment:
Cloud Agent allows you to run Tracetest tests against public facing endpoints without deploying the Tracetest Agent. This allowing you to trigger tests against public endpoints and collect traces information from either public trace data stores or, alternatively, directly receive incoming OTLP data.